utils/test/jest.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   testEnvironment: 'jest-environment-jsdom',
   setupFilesAfterEnv: ['<rootDir>/utils/test/jest.setup.js'],
   modulePathIgnorePatterns: ['./node_modules'],
-  transformIgnorePatterns: ['\\/node_modules\\/(?!@zendeskgarden|@babel)'],
+  transformIgnorePatterns: ['/node_modules/(?!(@zendeskgarden|@babel|react-merge-refs))'],
```

Comment:
I'm curious what made these `jest.config.js` changes necessary? We've been using `react-merge-refs` throughout the component codebase.

Reply:
It's because jest isn't converting `react-merge-refs` to commonjs. But that's a good point, I'll see why this works in `react-components` but not here.

Reply:
Looks like `react-merge-refs` is on v1.x in `react-components`, which is why it works there. I guess in v2.x the maintainer switched to `.mjs` which jest doesn’t like.

Reply:
Removed `babel-jest` and configured `swc/jest` to handle `react-merge-refs`. we still need the `transformIgnorePatterns` change since the package exports an es module.

Reply:
Yes this is still needed if no CJS version is present for a `node_modules` package used within `jest` tests.

Reply:
Still all needed now that `react-merge-refs` is only part of Storybook?